build.gradle

```diff
@@ -289,6 +337,17 @@ dependencies {
   testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+
+    contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5', version: diuspactproviderVersion
+    contractTestCompile group: 'au.com.dius.pact.provider', name: 'spring', version: diuspactproviderVersion
+    contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5spring', version: diuspactproviderVersion
+    contractTestRuntime group: 'io.springfox', name: 'springfox-boot-starter', version: swagger2Version
+    contractTestCompile group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-resource-server', version: '2.5.5'
+    contractTestCompile group: 'org.springframework.security', name: 'spring-security-oauth2-client', version: springsecurityVersion
+    contractTestCompile group: 'org.springframework.security', name: 'spring-security-oauth2-resource-server', version: springsecurityVersion
+    contractTestCompile group: 'org.springframework.security', name: 'spring-security-oauth2-core', version: springsecurityVersion
+    contractTestCompile group: 'org.springframework.security', name: 'spring-security-oauth2-jose', version: springsecurityVersion
+
```

Comment:
All of these dependencies are **inside** the `{}` relating to lines 337-351. They should be outside it.

Also I think we should initially only need the first 3 which relate to PACT.